phys2bids/viz.py

```diff
@@ -42,6 +49,10 @@ def plot_trigger(time, trigger, fileprefix, tr, thr, num_timepoints_expected,
     Outcome:
     fileprefix + _trigger_time.png:
         Creates new plot `fileprefix_trigger_time.png`.
+    See Also
+    --------
+    https://phys2bids.readthedocs.io/en/latest/howto.html
+    https://matplotlib.org/
```

Comment:
```suggestion
    matplotlib.pyploy.figsize
```

Reply:
numpy docstrings lets you "link" different functions and libraries in a sort of smart way, by just declaring the function. IN some programs, if it's imported, it will also redirect to its help!